src/CentreonRemote/Application/Clapi/CentreonRemoteServer.php

```diff
@@ -48,6 +48,10 @@ public function enableRemote($ip)
         $result = $this->getDi()['centreon.db-manager']->getRepository(InformationsRepository::class)->authorizeMaster($ip);
         echo 'Done'. "\n";
 
+        echo "\n Set 'remote' instance type...";
+        system("sed -i -r 's/central/remote/g' " . _CENTREON_ETC_ . "/conf.pm");
```

Comment:
Sed should be more restrictive as the word "central" can be used for password or database name or anything else.